autogpt/poetry.lock

```diff
@@ -355,7 +355,7 @@ uvicorn = "^0.23.2"
 webdriver-manager = "^4.0.1"
 
 [package.extras]
-benchmark = ["agbenchmark @ file:///home/reinier/code/agpt/Auto-GPT/benchmark"]
+benchmark = ["agbenchmark @ file:///Users/czerwinski/Projects/AutoGPT/benchmark"]
```

Comment:
I didn't realize the absolute path was saved in the `poetry.lock` :')

Reply:
We may want to switch to remote dependency